### ISSUE TITLE
Doens't work with the new gmail look

### ISSUE THREAD

**Issue by JStyle21** [gmail-manager-community/gmail-manager-community#60]
Gmail just changed to the new look and now this extension is broken.

**Comment by dunharg** [comment]
Confirmed. I can occasionally get it to work by re-entering the password, but it doesn't stick for more than a couple of days.

**Comment by adomj1** [comment]
Yes, I have also updated the Gmail latest version. It is working really well. Initially, I have faced some problem to log in but after taking help from [Optimum Customer Service](https://www.emailcustomercareservice.com/optimum-email-customer-service/). It gets resolved. If you have any problem with Gmail simply contact them.
